<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizSection</class>
 <widget class="QWidget" name="QuizSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>549</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QLabel" name="quizLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: white;
	color: #ada580;
	border-bottom: 2px solid #3b4135;
}</string>
     </property>
     <property name="text">
      <string>‚Äï QUIZ ‚Äï</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="beginButton">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>440</y>
       <width>550</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #3b4135;
	color: #ffe54d;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #5d6754;
}

QPushButton:pressed {
	background-color: #808d73;
}</string>
     </property>
     <property name="text">
      <string>BEGIN</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="instructions">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>110</y>
       <width>550</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
	border: 1px solid #3b4135;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Instructions&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:700;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- Each quiz question will take the form of an image or a video of a cat. Your job is to correctly identify the behavior the cat is displaying.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- To answer questions, you're given a screen with a small mouse, and three holes in the &amp;quot;wall&amp;quot;, above which are three possible answers. Using the WASD keys, move the mouse so it enters the hole corresponding to the correct answer. Make sure to avoid traps and collect cheese!&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- For every question you answer correctly, rewards for your mouse will increase&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- Answer all 12 questions correctly to win!&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Recommendations&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- Make sure to have your audio turned &lt;span style=&quot; font-weight:700;&quot;&gt;on&lt;/span&gt; and turned &lt;span style=&quot; font-weight:700;&quot;&gt;up&lt;/span&gt;. Some questions asks about sounds that a cat is making.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- Pay attention! You can't review an image or video after progressing to the answer screen.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Click the 'Begin' button to start. Good luck!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="videoQuestionHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: white;
	color: #ada580;
	border-bottom: 2px solid #3b4135;
}</string>
     </property>
     <property name="text">
      <string>‚Äï VIDEO QUESTION ‚Äï</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="videoQuestionText">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>80</y>
       <width>350</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>What behavior is this cat displaying?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QVideoWidget" name="videoWidget" native="true">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>460</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startVideoButton">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>460</y>
       <width>50</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 5px;
}

QPushButton:enabled {
	background-color: #3b4135;
	color: #ffe54d;
}

QPushButton:enabled:hover {
	background-color: #5d6754;
}

QPushButton:enabled:pressed {
	background-color: #808d73;
}

QPushButton:disabled {
	background-color: #bdbdbd;
	color: #9f9f9f;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>‚èµ</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="videoAnswerButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>490</y>
       <width>145</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 5px;
}

QPushButton:enabled {
	background-color: #3b4135;
	color: #ffe54d;
}

QPushButton:enabled:hover {
	background-color: #5d6754;
}

QPushButton:enabled:pressed {
	background-color: #808d73;
}

QPushButton:disabled {
	background-color: #bdbdbd;
	color: #9f9f9f;
}</string>
     </property>
     <property name="text">
      <string>Answer Question ü°Ü</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="imageAnswerButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>490</y>
       <width>145</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #3b4135;
	color: #ffe54d;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #5d6754;
}

QPushButton:pressed {
	background-color: #808d73;
}</string>
     </property>
     <property name="text">
      <string>Answer Question ü°Ü</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageQuestionHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: white;
	color: #ada580;
	border-bottom: 2px solid #3b4135;
}</string>
     </property>
     <property name="text">
      <string>‚Äï IMAGE QUESTION ‚Äï</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageQuestionText">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>80</y>
       <width>350</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>What behavior is this cat displaying?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>460</width>
       <height>330</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Goes Here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QuizGame" name="gameScreen" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>549</height>
      </rect>
     </property>
     <widget class="QPushButton" name="gameContinue">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>145</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #3b4135;
	color: #ffe54d;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #5d6754;
}

QPushButton:pressed {
	background-color: #808d73;
}</string>
      </property>
      <property name="text">
       <string>Continue (TEMP)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QuizSimulation" name="simulationScreen" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <property name="styleSheet">
     <string notr="true">QWidget#page_6 {
	background: url(:/images/end_screen_background.jpg);
}</string>
    </property>
    <widget class="QLabel" name="congratsLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
}</string>
     </property>
     <property name="text">
      <string>CONGRATS!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="scoreLabel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>120</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;
}</string>
     </property>
     <property name="text">
      <string>You finished with 0 points.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>533</x>
       <y>200</y>
       <width>175</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #faf2db;
	color: #db6b77;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #e0cca7;
}

QPushButton:pressed {
	background-color: #bbb18e;
}</string>
     </property>
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QuizSimulation</class>
   <extends>QWidget</extends>
   <header>quizsimulation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QuizGame</class>
   <extends>QWidget</extends>
   <header location="global">quizgame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
